internal/getter/hub.go

```diff
@@ -14,7 +14,7 @@ func (h HubDetector) Detect(src, pwd string) (string, bool, error) {
 	if len(src) == 0 {
 		return "", false, nil
 	}
-	fileDetector := fileDetector{}
+	fileDetector := NewFileDetector(false)
```

Comment:
We pass `false` here otherwise the file detector will detect `aws` as a file (for example), so we let the hub detector run in that case.

Reply:
```
policy aws {
  source = "./some/path/to/policy/aws"
}
```
Would this work?

Reply:
Let me verify that use case.

The logic for `hub.go` doesn't change (only for the file detector), but I'll verify this works.